src/Utils.cpp

```diff
@@ -36,6 +37,14 @@ vector<ITransformation *> *getTransformationsFromYaml(const string &config_path)
                 statements->push_back(stmt.as<string>());
             }
             transformations->push_back(new AddCommentsTransformation(p, statements));
+        } else if (transform_data["rename entities"]) {
+            auto p = transform_data["rename entities"]["p"].as<float>();
+            auto seed = transform_data["rename entities"]["seed"].as<int>();
+            const auto entities = new vector<string>;
+            for (auto ent : transform_data["rename entities"]["entities to rename"]) {
+                entities->push_back(ent.as<string>());
+            }
+            transformations->push_back(new RenameEntitiesTransformation(p, entities, seed));
```

Comment:
Since there are only two options for renaming let's use flags to choose renaming entities. I think it would be more efficient and accurate. We can use `rename functions` and `rename variables` with default false values (if not specified than keep old names).